docs/exceptionTelemetry.md

```diff
@@ -14,7 +14,10 @@ const appInsights = new ApplicationInsights({
     config: {
         connectionString: 'InstrumentationKey=YOUR_INSTRUMENTATION_KEY_GOES_HERE',
         expCfg: {
-            inclScripts: true
+            inclScripts: true,
+            expLog : () => {
+                return {message: ["log info 1", "log info 2"], maxLength: 100};
```

Comment:
The maxLength should be it's own config at the same level as the inclScripts and expLog, and I was thinking that it would just return a string []. ( `expLog: () => string[]` ), but you pose an interesting option....

If we make this return an "object" instead, then it provides for some level of extension without introducing a new function....

Maybe
```ts
interface IExceptionConfig: {
    inclScripts: boolean,
    expLog: () => { logs: string[] }
    maxLogs: number
}
```

Where the expLog function can return the logs wrapped in the object and we can add "new" optional properties based on feedback / requests.
```ts
function getExpLogs() {
    return {
        logs: ["log info 1", "log info 2"]
    };
}

extCfg: {
    inclScripts: true,
    expLog: getExpLogs,
    maxLogs: number
}

```